### ISSUE TITLE
Automatic Battery Saving

### ISSUE THREAD

**Issue by user44000** [elastic-rock/Candle#22]
Can you add an option for an automatic battery saving mode when the battery saver is activated? The setting could temporarily disable keep screen on and the flickering light.

**Comment by elastic-rock** [comment]
Could you, please, elaborate on the purpose of such a feature? It would be great if you could explain how you currently use the app and how the feature would benefit you.

**Comment by user44000** [comment]
I was thinking that the app should have a battery saving mode for if the devices battery is low so it drains less battery when the device is low on battery. The battery saving mode could automatically activate when the system battery saver is enabled. There could be a setting to enable or disable the feature.

**Comment by elastic-rock** [comment]
Sorry, I did not express myself clearly. Let me rephrase...

How do you currently use the app and how would the feature enhance your workflow?

**Comment by user44000** [comment]
I use this app on my phone and tablet. I don't usually use the app when my device is low on battery. I prefer to leave the keep screen on option enabled and I was thinking that a battery saver mode could be used to temporarily disable this setting without needing to disable it manually when the battery is low.

**Comment by elastic-rock** [comment]
This is not something I plan on working at the moment. Maybe in the future... 🙂
